### ISSUE TITLE
Manifest is not detected on Chrome

### ISSUE THREAD

**Issue by kms0219kms** [ShafSpecs/remix-pwa#154]
I configured the manifest like this: https://github.com/lunaiz-corp/am-to-mxm/blob/f864c2f846dc7f6d30f2c6a33ccfd266f57844a0/apps/frontend/app/routes/manifest%5B.webmanifest%5D.ts

And my browser (Chrome says)
![image](https://github.com/user-attachments/assets/64fff5c0-738c-4a3a-828b-e56fc7dc4c3b)

--------

I checked the specification from this website: https://web.dev/articles/add-manifest and I got the problem -
<link rel="webmanifest" href="/manifest.webmanifest"/>
This link tag's rel is should be manifest, not webmanifest.

**Comment by ShafSpecs** [comment]
The rel should not be the issue. Btw, webmanifest is now being preferred to manifest.

I think the issue is that you would have to pass the manifestUrl prop manually. For some reason, it doesn't register the implicit path. This is something I'm fixing ahead of the next update

**Comment by kms0219kms** [comment]
@ShafSpecs Not really. I just changed webmanifest to manifest in rel, then it worked.
It should be the issue with rel property. Thanks

**Comment by ShafSpecs** [comment]
> @ShafSpecs Not really. I just changed webmanifest to manifest in rel, then it worked.
> It should be the issue with rel property. Thanks

Thank you for the tip 😄

**Comment by jarrisondev** [comment]
Same problem.  Thanks for the solution. @kms0219kms

**Comment by rifont** [comment]
First of all, thanks for the awesome package! It made integrating my app with offline capabilities a breeze 👏

I also encountered this issue where the web manifest was not successfully detected by the browser on Chrome, and Safari iOS didn't allow installation of PWA with `standalone` mode. Following the suggestion to change `rel="webmanifest"` -> `rel="manifest"` resolved the issue.

[MDN docs](https://developer.mozilla.org/en-US/docs/Web/Manifest#deploying_a_manifest) and [Web dev](https://web.dev/articles/add-manifest#link-manifest) docs both list `rel="manifest"` as the correct spec to follow.

> The rel should not be the issue. Btw, webmanifest is now being preferred to manifest.
@ShafSpecs , could you please clarify the source for this claim?

I am happy to raise a PR to resolve this issue in the `<ManifestLink />` component and update the [Web Manifest docs](https://remix-pwa.run/docs/latest/web-manifest) to include links to the MDN & Web.dev docs, it will be useful context. Furthermore I suggest adding `rel` as a prop to the `<ManifestLink />` component to allow for customization as needed.

**Comment by ShafSpecs** [comment]
Hello @rifont 👋,

This issue is to be solved in the next major release of Remix PWA, it's already out via the dev release of the manifest package. The main thing holding up the next release now is react router support
